pages/datasets/_datasetId.vue

```diff
@@ -187,26 +187,32 @@ export default {
 
   mixins: [Request, DateUtils, FormatStorage],
 
-  asyncData() {
-    return Promise.all([
-      // Get page content
-      client.getEntries({
-        content_type: process.env.ctf_organ_id
-      })
-    ])
-      .then(([entries]) => {
-        return {
-          entries: entries.items
-        }
-      })
-      .catch(console.error)
+  async asyncData({ route, $axios }) {
+    const organEntries = await client.getEntries({
+      content_type: process.env.ctf_organ_id
+    })
+
+    const datasetId = pathOr('', ['params', 'datasetId'], route)
+    const datasetUrl = `${process.env.discover_api_host}/datasets/${datasetId}`
+    const simulationUrl = `${process.env.portal_api}/sim/dataset/${datasetId}`
+
+    const datasetDetails = await $axios.$get(datasetUrl)
```

Comment:
You're calling to get the dataset AND the simulation? Would you be able to get the dataset OR the simulation based on the query param? It looks the simulation endpoint returns everything you need from the dataset endpoint.

Reply:
> Navigating to the simulation page, either directly or from the Find Data page, results in the following error:
> 
> ```
> 
>   | vue__WEBPACK_IMPORTED_MODULE_19__.default.config.errorHandler | @ | client.js?06a0:77
> -- | -- | -- | --
>   | globalHandleError | @ | utils.js?ebed:16
>   | _callee4$ | @ | client.js?06a0:463
>   | tryCatch | @ | runtime.js?96cf:62
>   | invoke | @ | runtime.js?96cf:296
>   | prototype.<computed> | @ | runtime.js?96cf:114
>   | asyncGeneratorStep | @ | asyncToGenerator.js?1da1:3
>   | _throw | @ | asyncToGenerator.js?1da1:29
>   | Promise.then (async) |   |  
>   | asyncGeneratorStep | @ | asyncToGenerator.js?1da1:13
>   | _next | @ | asyncToGenerator.js?1da1:25
>   | Promise.then (async) |   |  
>   | asyncGeneratorStep | @ | asyncToGenerator.js?1da1:13
>   | _next | @ | asyncToGenerator.js?1da1:25
>   | Promise.then (async) |   |  
>   | asyncGeneratorStep | @ | asyncToGenerator.js?1da1:13
>   | _next | @ | asyncToGenerator.js?1da1:25
>   | Promise.then (async) |   |  
>   | asyncGeneratorStep | @ | asyncToGenerator.js?1da1:13
>   | _next | @ | asyncToGenerator.js?1da1:25
>   | Promise.then (async) |   |  
>   | asyncGeneratorStep | @ | asyncToGenerator.js?1da1:13
>   | _next | @ | asyncToGenerator.js?1da1:25
>   | eval | @ | asyncToGenerator.js?1da1:32
>   | eval | @ | asyncToGenerator.js?1da1:21
>   | _render | @ | client.js?06a0:239
>   | render | @ | client.js?06a0:239
>   | iterator | @ | vue-router.esm.js?8c4f:2154
>   | step | @ | vue-router.esm.js?8c4f:1880
>   | eval | @ | vue-router.esm.js?8c4f:1881
>   | eval | @ | vue-router.esm.js?8c4f:2173
>   | _callee3$ | @ | client.js?06a0:160
>   | tryCatch | @ | runtime.js?96cf:62
>   | invoke | @ | runtime.js?96cf:296
>   | prototype.<computed> | @ | runtime.js?96cf:114
>   | Show 19 more frames
> ```

@cameronbaney can you provide steps to reproduce this issue? I'm not seeing the same error unfortunately.